packages/fetch-and-compile/test/fixture.js

```diff
@@ -0,0 +1,84 @@
+const fs = require("fs");
+const path = require("path");
+
+const mainnetData = {
+  status: "1",
+  message: "OK-Missing/Invalid API Key, rate limit of 1/5sec applied",
+  result: [
+    {
+      SourceCode: fs
+        .readFileSync(path.resolve(__dirname, "./sol/mainnet.sol"))
```

Comment:
A note on use of `readFileSync`: Rather than first doing `fs.readFileSync(path)` to read the file into a `Buffer` and then doing `buffer.toString()` to render it as a string, you can just do `fs.readFileSync(path, "utf8")` to directly read the file in as a string, which would hopefully avoid two passes over the file.